runtime/queries/filterutil.go

```diff
@@ -0,0 +1,88 @@
+package queries
+
+import (
+	"github.com/rilldata/rill/proto/gen/rill/runtime/v1"
+	"google.golang.org/protobuf/types/known/structpb"
+)
+
+func FilterColumn(col string) *runtimev1.Expression {
```

Comment:
Consider moving these utils to `runtime/pkg/expressionpb` and making the names match the `Expression` type (e.g. `expressionpb.NotIn` instead of `FilterNotInClause`).